### PR TITLE
Fix trackError parameters

### PR DESCRIPTION
**Pull request by Jack-Keene**
This PR fixes issue #28

### CODE REVIEW

File: template.tpl [hunk 1]
```diff
@@ -2530,6 +2523,13 @@ ___WEB_PERMISSIONS___
         "versionId": "1"
       },
       "param": [
+        {
+          "key": "allowedKeys",
+          "value": {
+            "type": 1,
+            "string": "specific"
+          }
+        },
```
Comment (jethron) [inline template.tpl:2532]: This seems to just be an update to the export format for this permission, not any actual change in the permissions as part of this change.

File: template.tpl [hunk 2]
```diff
@@ -1910,6 +1902,7 @@ switch (data.eventType) {
       return fail('Missing "message" from error tracking hit.');
 
     commandName = data.eventType;
+    parameters = paramObj;
```
Comment (jethron) [inline template.tpl:1905]: :+1:

If only a test had caught this earlier. :joy:

File: template.tpl [hunk 3]
```diff
@@ -34,34 +26,35 @@ ___TEMPLATE_PARAMETERS___
 
 [
```
Comment (jethron) [inline template.tpl:27]: After re-sorting the keys to compare, there aren't really any actual changes to the JSON introduced here.

File: template.tpl [hunk 4]
```diff
@@ -879,163 +885,162 @@ ___TEMPLATE_PARAMETERS___
...
             "valueHint": "class1,class2,class3"
           },
           {
-            "type": "TEXT",
-            "name": "linkTrackingDenylist",
+            "help": "Add a comma-separated list (or variable that returns an array) of HTML classes. If any of these classes is on the HTML element when a link click event is triggered, the event will not be tracked. NOTE! This option has no effect if Allowlist Classes is specified.",
             "displayName": "Denylist Classes",
             "simpleValueType": true,
-            "valueHint": "class1,class2,class3",
-            "help": "Add a comma-separated list (or variable that returns an array) of HTML classes. If any of these classes is on the HTML element when a link click event is triggered, the event will not be tracked. NOTE! This option has no effect if Allowlist Classes is specified.",
```
Comment (jethron) [inline template.tpl:976]: Trivia: this trailing comma meant this was actually not valid JSON before this update to the export format. Wonder how that happened.

**Review by jethron — APPROVED** [review]
LGTM after TOS agreement confirmation/fix. 

A test or something would be nice to prevent regressions but if it was there I'd probably want it for every command type so we'll just ignore that slippery slope for now.

File: template.tpl (hunk 3 above)
Comment (Jack-Keene) [reply template.tpl:27]: Yep - it should just be the one line change plus the format change in the export!

**Pull request merged**